test/unit/client-daemonset.bats

```diff
@@ -2,391 +2,6 @@
 
 load _helpers
 
-@test "client/DaemonSet: enabled by default" {
```

Comment:
Why were these tests deleted? Not sure I follow.

Reply:
...and that's why we do code review! 🤦 

I had deleted them so I could test my changes faster and then forgot.

Reply:
😅 lol..ive been there before